Qlc/Objects/Transaction.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Newtonsoft.Json;
+
+namespace Qlc.Objects
+{
+    public class Transaction
```

Comment:
add prefix Pov to all PoV related objects should be better.

Reply:
Done